subsys/bluetooth/controller/Kconfig

```
@@ -42,11 +42,18 @@ endchoice
 if BT_LL_SOFTDEVICE
 menu "SoftDevice Controller"
 
+config BT_LL_SOFTDEVICE_HEADERS_INCLUDE
+	bool
+	default y if !BT_LL_SOFTDEVICE
```

The headers should also be included when building as lib
```suggestion
	default y if !BT_LL_SOFTDEVICE || BT_LL_SOFTDEVICE_BUILD_TYPE_LIB
```